pkg/steps/template.go

```diff
@@ -570,7 +570,7 @@ func waitForPodCompletion(podClient coreclientset.PodInterface, name string, not
 					log.Printf("%v", err)
 				}
 			}
-			return fmt.Errorf("could not wait for pod completion: %v", err)
+			return fmt.Errorf("%v", err)
```

Comment:
Why not returning `err` straight?

```suggestion
			return err
```